accounts/fixtures/department.json

```diff
@@ -0,0 +1,30 @@
+[
+  {
+    "model": "accounts.department",
+    "pk": 1,
+    "fields": {
+      "department_name": "Software Engineering"
+    }
+  },
+  {
+    "model": "accounts.department",
+    "pk": 2,
+    "fields": {
+      "department_name": "Science, Technology & Society"
+    }
+  },
+  {
+    "model": "accounts.department",
+    "pk": 3,
+    "fields": {
+      "department_name": "Interaction Design"
+    }
+  },
+  {
+    "model": "accounts.department",
+    "pk": 4,
+    "fields": {
+      "department_name": "Product Management"
+    }
+  }
+]
```

Comment:
Here, I would add 'other' as an option. An example for other might be an English teacher or English learning books.